### PR TITLE
SEN-1231 & SEN-1236 -- jeckyl setup

### PR DESCRIPTION
**Pull request by noTxt**
**testing:**
- checkout `noTxt:SEN-1229-a`
- `bundle install` ( `gem install bundler` if you don't have bundler )
- `bower install`
- `bundle exec jekyll serve --drafts`
- visit [Local site](http://localhost:4000/) and inspect theme

**ticket:**
https://cascadeenergy.atlassian.net/browse/SEN-1231 & https://cascadeenergy.atlassian.net/browse/SEN-1236
epic - https://cascadeenergy.atlassian.net/browse/SEN-1227

**note:**
this content creation is delt with in:
https://cascadeenergy.atlassian.net/browse/SEN-1234
https://cascadeenergy.atlassian.net/browse/SEN-1235

### CODE REVIEW

File: src/_data/team.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+- name: michael
+  github: noTxt
+
+- name: will
+  github: nackjicholson
+
+- name: jinsy
+  github: jinsyaoommen
+
+- name: nick
```
Comment (cberube) [inline src/_data/team.yml]: Can we capitalize the names in the link list? Feels odd to have them lowercase.

**Comment by cberube** [comment]
I had to do a handful of additional setup steps to get `bundle install` to stop complaining:
* `sudo yum install ruby-devel`
* `sudo gem install RedCloth -v '4.2.9'`
* `sudo yum install patch`
* `sudo gem install nokogiri -v '1.6.6.2'`

File: src/_data/team.yml (hunk 1 above)
Comment (noTxt) [inline src/_data/team.yml]: - [ ] captitalize names

File: src/_data/stack.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  link: https://angularjs.org/
+
+- caption: nodejs
+  link: https://nodejs.org/
+
+- caption: lambda
+  link: http://aws.amazon.com/lambda/
+
+- caption: dynamodb
+  link: http://aws.amazon.com/dynamodb/
```
Comment (cberube) [inline src/_data/stack.yml]: We should fix the capitalization and naming here to match what each service does as well -- so these should be:
* `Bootstrap`
* `AngularJS`
* `Node.js`
* `AWS Lambda`
* `Amazon DynamoDB`

File: src/_data/cool.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+- caption: mutton
+  link: https://github.com/CascadeEnergy/mutton
+
+- caption: marshaler
```
Comment (cberube) [inline src/_data/cool.yml]: This should read `dynamodb-marshaler`, to match the package name

**Comment by nackjicholson** [comment]
:ok: :shipit:

File: src/_data/stack.yml (hunk 2 above)
Comment (noTxt) [inline src/_data/stack.yml]: - [ ] capitalize

File: src/_data/cool.yml (hunk 3 above)
Comment (noTxt) [inline src/_data/cool.yml]: - [ ] use package name

File: src/_data/philosophies.yml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+- caption: clean coder
```
Comment (cberube) [inline src/_data/philosophies.yml]: `The Clean Coder`

File: src/_includes/links.html [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    <div class="panel-heading">our stack</div>
+    <div class="list-group">
+        {% for item in site.data.stack %}
+            <a href="{{ item.link }}" class="list-group-item">{{ item.caption }}</a>
+        {% endfor %}
+    </div>
+</div>
+
+<div class="panel panel-default">
+    <div class="panel-heading">( just plain cool )</div>
```
Comment (cberube) [inline src/_includes/links.html]: Can we brainstorm another title for this section? Not sure what exactly, but something more like `Cascade Open Source`, `Cascade Tools`, `Custom Tools`, etc?

File: src/_data/philosophies.yml (hunk 4 above)
Comment (noTxt) [inline src/_data/philosophies.yml]: - [ ] capitalize

File: src/_includes/nav.html [hunk 6]
```diff
@@ -0,0 +1,12 @@
+<nav class="navbar">
+    <div class="container-fluid">
+        <div class="navbar-header">
+            <a class="navbar-brand" href="/">
+                <img alt="Brand" src="/img/logo.png">
+            </a>
+        </div>
```
Comment (cberube) [inline src/_includes/nav.html]: Can we get the name `Cascade Energy` in the navbar near the logo and switch the about link to read `About us`?

File: src/_layouts/post.html [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+    <div class="col-xs-12 col-sm-11 col-lg-9">
+        <article class="uk-article">
+            <h1 class="uk-article-title">{{ page.title }}</h1>
+            <p class="uk-article-meta">Written by {{ page.author }} on {{ page.date | date_to_long_string }}.</p>
+            {{ content }}
+        </article>
+    </div>
+    <div class="col-lg-2 hidden-xs hidden-sm hidden-md">
+        {% include links.html %}
+    </div>
```
Comment (cberube) [inline src/_layouts/post.html]: Do the links just disappear when the screen size is small? I think we'd want them visible at all sizes, even if they flip to below the content.

File: src/_includes/links.html (hunk 5 above)
Comment (noTxt) [inline src/_includes/links.html]: I'd say let's pick something we can live with and tweak it later, I like `Cascade Open Source`

- [ ] rename

File: src/_layouts/post.html (hunk 7 above)
Comment (noTxt) [inline src/_layouts/post.html]: yeah, the links seem like decoration to me, I wanted it to strip down into just an article at small form factor

- [ ] keep links visible but move them below on small form factor

File: src/_includes/nav.html (hunk 6 above)
Comment (noTxt) [inline src/_includes/nav.html]: - [ ] rename to `About Us`
- [ ] Add `Cascade Energy`

Comment (noTxt) [inline src/_includes/nav.html]: It's not looking very good with `Cascade Energy` written. It's already text heavy and feels cluttered

screen shot 2015-03-11 at 12 09 55 pm
vs.
screen shot 2015-03-11 at 12 10 15 pm

**Comment by cberube** [comment]
@noTxt -- I really had less changing of the header in mind:
screen shot 2015-03-11 at 12 21 59 pm

I think it's a more complicated problem if you want to move the logo around.

**Comment by noTxt** [comment]
blerg ... I reset this branch after tweaking the layout and it got all messed up

**Pull request closed without merging**